### PR TITLE
Error correction

### PR DESCRIPTION
**Pull request by rpassas**
### Summary

- changed DeathCertificate entries to `composition/...` in the DeathCertificateDocument examples
- added rule set for absolute references where needed in examples
- added to ignore warnings

### CODE REVIEW

File: input/fsh/Ex_Messaging.fsh [hunk 1]
```diff
@@ -65,7 +69,7 @@ Description: "Header for Update: Example"
 // * eventUri = "http://nchs.cdc.gov/vrdr_submission_update"
 * destination.endpoint = "http://nchs.cdc.gov/vrdr_submission"
 * source.endpoint = "https://sos.ny.gov/vitalrecords"
-* focus.reference = "Bundle/DeathCertificateDocument-Example1"
+* focus.reference = "http://www.example.org/fhir/Bundle/DeathCertificateDocument-Example1"
```
Comment (saulakravitz) [inline input/fsh/Ex_Messaging.fsh]: Any reason these shouldn't all be like this for consistency?
insert addReference(focus, Bundle, DeathCertificateDocument-Example1)

File: input/fsh/Ex_Messaging.fsh [hunk 2]
```diff
@@ -74,7 +78,7 @@ Description: "Header for Submission - Example"
 // * eventUri = "http://nchs.cdc.gov/vrdr_submission"
 * destination.endpoint = "http://nchs.cdc.gov/vrdr_submission"
 * source.endpoint = "https://sos.ny.gov/vitalrecords"
-* focus.reference = "Bundle/DeathCertificateDocument-Example1"
+* focus.reference = "http://www.example.org/fhir/Bundle/DeathCertificateDocument-Example1"
```
Comment (saulakravitz) [inline input/fsh/Ex_Messaging.fsh]: Use ruleset for consistency?

File: input/fsh/Ex_Messaging.fsh (hunk 1 above)
Comment (rpassas) [reply input/fsh/Ex_Messaging.fsh]: no reason, just an oversight. I was testing it out as a sanity check and forgot to change it. Thanks for catching

**Review by saulakravitz — CHANGES_REQUESTED** [review]
I think these info/warnings are worth fixing, since they probably indicate real problems:

INFORMATION: Bundle/CauseOfDeathCodingMessage-Example1: Bundle: Can't find 'Patient/Decedent-Example1' in the bundle (Bundle.entry[2].resource.entry[10].resource.subject)
INFORMATION: Bundle/CauseOfDeathCodingMessage-Example1: Bundle: Can't find 'Practitioner/Certifier-Example1' in the bundle (Bundle.entry[2].resource.entry[13].resource.performer[0])
INFORMATION: Bundle/CauseOfDeathCodingUpdateMessage1: Bundle: Can't find 'Patient/Decedent-Example1' in the bundle (Bundle.entry[2].resource.entry[10].resource.subject)
INFORMATION: Bundle/CauseOfDeathCodingUpdateMessage1: Bundle: Can't find 'Practitioner/Certifier-Example1' in the bundle (Bundle.entry[2].resource.entry[13].resource.performer[0])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Patient/Decedent-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.subject)
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Practitioner/Certifier-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.author[0])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Procedure/DeathCertification-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.event[0].detail[0])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'RelatedPerson/DecedentFather-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[1])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'RelatedPerson/DecedentMother-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[2])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'RelatedPerson/DecedentSpouse-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[3])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/DecedentAge-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[4])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/BirthRecordIdentifier-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[5])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/DecedentEducationLevel-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[6])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/DecedentMilitaryService-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[7])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/DecedentUsualWork-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[8])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/InputRaceAndEthnicity-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[9])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/EmergingIssues-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[10])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/ExaminerContacted-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[1].entry[0])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/DecedentPregnancyStatus-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[1].entry[1])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/TobaccoUseContributedToDeath-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[1].entry[2])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/AutopsyPerformedIndicator-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[1].entry[3])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Location/DeathLocation-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[1].entry[4])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Location/InjuryLocation-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[1].entry[5])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/InjuryIncident-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[1].entry[6])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/DeathDate-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[1].entry[7])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/SurgeryDate-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[1].entry[8])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/MannerOfDeath-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[2].entry[2])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/CauseOfDeathPart1-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[2].entry[3])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/CauseOfDeathPart1-Example2' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[2].entry[4])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/CauseOfDeathPart2-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[2].entry[5])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Location/DispositionLocation-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[3].entry[0])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Organization/FuneralHome-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[3].entry[1])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Observation/DecedentDispositionMethod-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[3].entry[2])
INFORMATION: Bundle/DeathRecordSubmissionMessage-Example1: Bundle: Can't find 'Practitioner/Mortician-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[3].entry[3])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Patient/Decedent-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.subject)
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Practitioner/Certifier-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.author[0])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Procedure/DeathCertification-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.event[0].detail[0])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'RelatedPerson/DecedentFather-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[1])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'RelatedPerson/DecedentMother-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[2])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'RelatedPerson/DecedentSpouse-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[3])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/DecedentAge-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[4])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/BirthRecordIdentifier-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[5])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/DecedentEducationLevel-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[6])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/DecedentMilitaryService-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[7])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/DecedentUsualWork-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[8])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/InputRaceAndEthnicity-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[9])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/EmergingIssues-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[0].entry[10])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/ExaminerContacted-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[1].entry[0])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/DecedentPregnancyStatus-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[1].entry[1])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/TobaccoUseContributedToDeath-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[1].entry[2])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/AutopsyPerformedIndicator-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[1].entry[3])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Location/DeathLocation-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[1].entry[4])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Location/InjuryLocation-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[1].entry[5])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/InjuryIncident-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[1].entry[6])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/DeathDate-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[1].entry[7])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/SurgeryDate-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[1].entry[8])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/MannerOfDeath-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[2].entry[2])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/CauseOfDeathPart1-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[2].entry[3])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/CauseOfDeathPart1-Example2' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[2].entry[4])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/CauseOfDeathPart2-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[2].entry[5])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Location/DispositionLocation-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[3].entry[0])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Organization/FuneralHome-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[3].entry[1])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Observation/DecedentDispositionMethod-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[3].entry[2])
INFORMATION: Bundle/UpdateMessage-Example1: Bundle: Can't find 'Practitioner/Mortician-Example1' in the bundle (Bundle.entry[2].resource.entry[0].resource.section[3].entry[3])

I updated ignorewarnings to block the uninteresting messages.

**Pull request closed without merging**